src/SamplesApp/SamplesApp.UITests/TestFramework/ActivePlatformsAttribute.cs

```diff
@@ -26,6 +27,27 @@ public Platform[] Platforms
 			}
 		}
 
+		public Exception Error { get; }
+
+		public ActivePlatformsAttribute(Platform platform, [CallerArgumentExpression("platform")] string expression = "")
+		{
+			// Errors out platform shouldn't be thrown here.
+			// We must assign to an Error property and throw in SampleControlUITestBase.BeforeEachTest (specifically, the GetActivePlatforms call).
+			// Otherwise, NUnit skips the test completely when it tries to load the attributes.
+			if (expression.Length == 0)
+			{
+				Error = new NotSupportedException("'CallerArgumentExpression' is not supported?");
+			}
+
+			if (expression.Contains('|'))
+			{
+				// See https://github.com/unoplatform/uno/issues/9798 for info.
+				Error = new ArgumentException("Don't use bitwise or in ActivePlatforms attribute.");
+			}
+
+			Properties.Add("ActivePlatforms", new[] { platform });
+		}
```

Comment:
If https://github.com/dotnet/runtime/issues/75412 was approved and implemented. We can remove this code and rely on the analyzer from `Microsoft.CodeAnalysis.NetAnalyzers` package